core/client/src/main/java/alluxio/hadoop/ConfUtils.java

```diff
@@ -84,6 +80,6 @@ public static Configuration loadFromHadoopConfiguration(
       }
     }
     LOG.info("Loading Alluxio properties from Hadoop configuration: {}", alluxioConfProperties);
-    return Configuration.fromMap(alluxioConfProperties);
+    Configuration.merge(alluxioConfProperties);
```

Comment:
I've created https://alluxio.atlassian.net/browse/ALLUXIO-2034 to track support for multiple clients with their own configuration. Could you add a TODO and link it here, and add a comment that we assume here there is only one client configuration in the JVM?